includes/Core/Storage/Setting_With_Owned_Keys_Trait.php

```diff
@@ -24,14 +26,15 @@ trait Setting_With_Owned_Keys_Trait {
 	 *
 	 * @since n.e.x.t
 	 *
-	 * @param array $keys An array of keys to check.
+	 * @param array  $keys An array of keys to check.
+	 * @param string $ownerID_key Optional. A key to use for storing owner ID setting.
```

Comment:
The descriptions should also be aligned; unfortunately PHPCS does not enforce this yet. Using the new name for the second parameter:
```suggestion
	 * @param array  $keys         An array of keys to check.
	 * @param string $owner_id_key Optional. The key to use for storing the owner ID.
```

Reply:
Updated.